ui-tests/src/test/java/org/openmrs/reference/page/PatientVisitsDashboardPage.java

```diff
@@ -60,12 +59,10 @@ public WebElement getActiveVisit() {
         return findElements(VISIT_LIST);
     }
 
-    public PatientVisitsDashboardPage endVisit() {
```

Comment:
Why endVisit() method is refactored ? 
 Two elements within the method can not be retrieved `VISIT_LIST` which can only be retrieved at `LN-`[44](https://github.com/openmrs/openmrs-distro-referenceapplication/blob/f32ade42c62bf5f39ea2237660efff3164a4d2af/ui-tests/src/test/java/org/openmrs/reference/EndVisitTest.java#L44) and `END_VISIT_DIALOG` which does not exist on the page

Its from the reason above that the method has been refactored to a return type of void. Void type by its implementation returns the test encounter to `PatienetVisistDashboardPage` as it was before when the return type was  PatienetVisistDashboardPage.